### ISSUE TITLE
Empty notifications area should have focus on the action toolbar

### ISSUE THREAD

**Issue by isidorn** [microsoft/vscode#97513]
Found by @jvesouza

1. Have no notifications, focus in editor, press F6
2. Move focus via arrows until notifications area is focused
3. Press `space` - focus is moved to the editor 🐛 

Focus should be in the notifications area, not in the editor.
Focus jumping to the editor is a bit sudden imho

**Comment by bpasero** [comment]
I suggest in the case of an empty notification center, focus should be on the action that collapses the notifications center again. The triangle down thing.

**Comment by isidorn** [comment]
Makes sense. Let's keep it open for a PR and if nobody jumps on it in a couple of milestones I can also look into it.
Thanks

**Comment by ChrisPapp** [comment]
Can I work on this?

**Comment by bpasero** [comment]
I think there is actually a bug. Pressing `Space` should not type into the editor.

**Comment by isidorn** [comment]
@ChrisPapp thanks for offering, however @bpasero and me just discussed this and came up with a very practical solution. Executing commands from the status bar no longer focuses the editor, this was some old behavior.
